### PR TITLE
Master v1.0.0a1

### PR DESCRIPTION
**Pull request by lnielsen**
Includes PR #2

### CODE REVIEW

File: invenio_config/utils.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+from .default import InvenioConfigDefault
+from .env import InvenioConfigEnvironment
+from .folder import InvenioConfigInstanceFolder
+from .module import InvenioConfigModule
+
+
+def create_conf_loader(config, env_prefix='APP'):
+    """Create a default configuration loader.
+
+    A configuration loader takes an Flask application and keyword arguments and
```
Comment (nharraud) [inline invenio_config/utils.py]: `a Flask application` not `an`

File: invenio_config/utils.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    updates the Flask application's configuration as it sees fit.
+
+    This default configuration loader will load configuration in the following
+    order:
+
+        1. Load configuration from ``config`` module provided as argument.
+        2. Load configuration from the instance folder:
+           ``<app.instance_path>/<app.name>.cfg``.
+        3. Load configuration keyword arguemnts provided.
```
Comment (nharraud) [inline invenio_config/utils.py]: `s/arguemnts/arguments/`

File: invenio_config/utils.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        4. Load configuration from environment variables with the prefix
+           ``env_prefix``.
+
+    If no secret key has been set a warning will be issued.
+
+    :param config: Either an import string to a module with configuration or
+        alternatively the module itself.
+    :param env_prefix: Environment variable prefix to import confirguration
```
Comment (nharraud) [inline invenio_config/utils.py]: s/confirguration/configuration/

File: invenio_config/utils.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        from.
+    :return: A callable with the method signature
+        ``conf_loader(app, **kwargs)``.
+    """
+    def _conf_loader(app, **kwargs_config):
+        if config:
```
Comment (nharraud) [inline invenio_config/utils.py]: The doc seems to say that `config` should always be set but this tests seems to show the contrary. If it is nullable we can put in the doc.

Comment (lnielsen) [inline invenio_config/utils.py]: Fixed. Changed into keyword argument and adjusted documentation slightly.

**Pull request merged**